Mage/src/main/java/mage/game/combat/CombatGroup.java

```diff
@@ -544,9 +549,14 @@ private void multiAttackerDamage(boolean first, Game game) {
 
     private void defenderDamage(Permanent attacker, int amount, Game game) {
         if (this.defenderIsPlaneswalker) {
-            Permanent defender = game.getPermanent(defenderId);
-            if (defender != null) {
-                defender.markDamage(amount, attacker.getId(), null, game, true, true);
+            Permanent planeswalker = game.getPermanent(defenderId);
+            if (planeswalker != null) {
+                //apply damage to player from TrampleOverPlaneswalkersAbility (for ThrastaTempestsRoar in set MH2)
```

Comment:
Try to use real card names in the comments, not classes (for best searching). Example:
```
// apply excess damage from "trample over planeswaslkers" ability (example: Thrasta, Tempest's Roar)
```